static/js/map/map.pokemon.js

```diff
@@ -54,6 +54,11 @@ function isPokemonMeetsFilters(pokemon, isNotifPokemon) {
         }
     }
 
+    
+    if(settings.filterPokemonBySize && settings.filterPokemonBySizeOptions && !settings.filterPokemonBySizeOptions.includes(pokemon.size)){
```

Comment:
On this line, can you put a space between the "if" and the "open parenthesis"?
And also, put a space between the last "close parenthesis" and the "open curly brace"?